helix-core/src/test/java/org/apache/helix/integration/task/TestEnqueueJobs.java

```diff
@@ -41,13 +47,39 @@
   public void beforeClass() throws Exception {
     setSingleTestEnvironment();
     super.beforeClass();
+    //WorkflowConfig.disableJobPurge();
+  }
+
+  @AfterClass
+  @Override
+  public void afterClass() throws Exception {
+    //WorkflowConfig.enableJobPurge();
+    super.afterClass();
+  }
+
+  @BeforeMethod
+  public void beforeTest(Method testMethod, ITestContext testContext) {
```

Comment:
Please confirm if this has already been done by the parent class ZkTestBase.
I think this is duplicate logic.

Reply:
Right. This should be removed.